book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -41,6 +41,28 @@ docker run --rm -it --privileged -p 4000:4000 ^
 wallaroo-labs-docker-wallaroolabs.bintray.io/{{ docker_version_url }}
 {%- endcodetabs %}
 
+## Shell 1: Start the Wallaroo Docker container for Machida3
+
+{% codetabs name="UNIX Bash", type="bash" -%}
+docker run --rm -it --privileged -p 4000:4000 \
+-v /tmp/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/wallaroo-src:/src/wallaroo \
+-v /tmp/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/python-virtualenv-machida3:/src/python-virtualenv \
```

Comment:
Should this `machida3` specific directory be used for non-`UNIX Bash` environments also?

Reply:
addressed